Examples/ConstructingPackets/Main.cs

```diff
@@ -36,6 +36,7 @@ public static void Main(string[] args)
 
             // and print out the packet to see that it looks just like we wanted it to
             Console.WriteLine(ethernetPacket.ToString());
+            Console.ReadKey(true);
```

Comment:
It was just to keep the window open to view the results of the example. I checked the other examples and they didn't have it, so I will change it to be consistent, and in case anyone decides to scrape it.